### ISSUE TITLE
No warning dereferencing field of default struct instance

### ISSUE THREAD

**Issue by cston** [dotnet/roslyn#32242]
Warnings should be reported dereferencing `S.F`:
```C#
#pragma warning disable 649
struct S
{
    internal object F;
}
class Program
{
    static void M()
    {
        S s = default;
        S t = new S();
        s.F.ToString(); // warning
        t.F.ToString(); // warning
    }
}
```

**Comment by cston** [comment]
Duplicate of #30731.
